src/main/webapp/i18n/de/error.json

```
@@ -94,6 +94,7 @@
         "unableToImportProgrammingExercise": "Fehler beim Importieren der Programmieraufgabe.",
         "titleAlreadyExists": "Es existiert bereits eine Programmieraufgabe mit demselben Titel, bitte nutze einen anderen Titel.",
         "shortnameAlreadyExists": "Es existiert bereits eine Programmieraufgabe mit demselben Kurznamen, bitte nutze einen anderen Kurznamen.",
-        "courseTitleTooLong": "Der Kurstitel ist zu lang (max. 255 Zeichen), bitte nutze einen kürzeren Kurstitel."
+        "courseTitleTooLong": "Der Kurstitel ist zu lang (max. 255 Zeichen), bitte nutze einen kürzeren Kurstitel.",
+        "notAllowedInExam": "Diese Operation ist in einer Klausur nicht erlaubt."
```

This sounds a bit strange to me, I don't think "operation" translates like that. Maybe use "Aktion" instead
```suggestion
        "notAllowedInExam": "Diese Aktion ist in einer Klausur nicht erlaubt."
```